### PR TITLE
ENH: Add Parsed Parameters in DICOMRequestHandler

### PR DESCRIPTION
**Pull request by stephencrowell**
Add PatientID to params parsed

### CODE REVIEW

File: Modules/Scripted/WebServer/WebServerLib/DICOMRequestHandler.py [hunk 1]
```diff
@@ -61,12 +61,16 @@ def handleStudies(self, parsedURL, requestBody):
...
         offset = 0
         limit = 100
+        patientID = None
         params = parsedURL.query.split(b"&")
         for param in params:
             if param.split(b"=")[0] == b"offset":
                 offset = int(param.split(b"=")[1])
             if param.split(b"=")[0] == b"limit":
                 limit = int(param.split(b"=")[1])
+            if param.split(b"=")[0] == b"PatientID":
```
Comment (lassoan) [inline Modules/Scripted/WebServer/WebServerLib/DICOMRequestHandler.py:71]: Let's accept 00100020, too:
```suggestion
            if param.split(b"=")[0] in [b"PatientID", b"00100020"]:
```

**Review by lassoan — COMMENTED** [review]
Thank you, it looks good to me (see just one minor improvement opportunity in the inline comment). Why did you make it a "Draft" pull request?

**Comment by stephencrowell** [comment]
> Thank you, it looks good to me (see just one minor improvement opportunity in the inline comment). Why did you make it a "Draft" pull request?

This may not be the only change to the `DICOMRequestHandler.py`. I am currently working on integrating DICOMweb functionality into [this](https://github.com/KitwareMedical/SlicerEHRSandbox) extension and using Slicer's WebServer is the easiest way to start up a DICOMweb server. I wanted to leave this PR up in case I need to make another adjustment.

I update the PR title to make it more clear there could be more changes than just PatientID

**Comment by lassoan** [comment]
> I wanted to leave this PR up in case I need to make another adjustment.

OK, as you prefer.

**Comment by stephencrowell** [comment]
@lassoan No rush, but wanted to ping you again in case you didn't see this/forgot over the holidays.

**Comment by ebrahimebrahim** [comment]
Is this ready to be merged?

**Comment by luzpaz** [comment]
Is there an ETA on this ?

**Review by lassoan — APPROVED** [review]
Thanks for your patience with this. I've added documentation, fixed paging (interpretation of offset and limit arguments), and tested it. It is good to go now.

**Pull request merged**